src/main/main.js

```diff
@@ -202,24 +209,11 @@ class Main {
 
     // Compile contents
     compile(fileContent, {})
-      .then(ast => {
+      .then(async ast => {
         // Get project URL if it exists
         const tokenPath = getTokenPath(p, this.workingDir);
 
-        let url = '';
-        try {
-          const token = fs.readFileSync(tokenPath, { encoding: 'utf-8' });
-          const tokenUrl = this.store.getTokenUrlByToken(token);
-          if (tokenUrl) {
-            url = tokenUrl.url;
-          } else {
-            // TODO: Get URL from idyll pub api here
-            console.log('Make request to server for url since token exists');
-            // url = await request.get({ url: IDYLL_PUB_API/{token} })
-          }
-        } catch (error) {
-          console.log(error, 'Token does not exist yet.');
-        }
+        let url = await requestUrl(tokenPath, this.store);
```

Comment:
Should we make this `const` instead of `let`?

Reply:
You right